azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -76,11 +76,17 @@ class SharedClientCreateFromConnectionStringTests(object):
     @pytest.mark.parametrize(
         "ca_cert",
         [
-            pytest.param(None, id="No CA certificate"),
-            pytest.param("some-certificate", id="With CA certificate"),
+            pytest.param(None, id=" No CA certificate "),
+            pytest.param("some-certificate", id=" With CA certificate "),
         ],
     )
-    def test_auth_provider_creation(self, mocker, client_class, connection_string, ca_cert):
+    @pytest.mark.parametrize(
+        "websockets",
+        [pytest.param(None, id=" No Websockets "), pytest.param(True, id=" With Websockets ")],
```

Comment:
> [pytest.param(None, id=" No Websockets "),  [](start = 7, length = 44)

is None equivalent to  False since False goes to default else block ?

Reply:
Yes. None is equivalent to False and also exercises that the default is used.